### PR TITLE
Node.js update

### PR DESCRIPTION
**Pull request by Shanzid01**
Contains the updated app.js and DBHelper.js with the variables needed to connect to the new MongoDB Atlas Database which is opened under the RSquared email.
Also contains the upozila and zila data cleaned json files (backup).

### CODE REVIEW

File: node.js_mongodb/DB_helper.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
 */
 
 const MongoClient = require('mongodb').MongoClient;
-const uri="mongodb+srv://common_user:xTp547a@r-squared-data-9sxhm.mongodb.net/region_data";
+const uri="mongodb+srv://common_user:xTp547a@rsquared-datacluster-te3yj.mongodb.net/rsquared_zilla_upazilla_db"; // <<-- new URI
```
Comment (azmainamin) [inline node.js_mongodb/DB_helper.js]: remove the comment. unnecessary.

File: node.js_mongodb/app.js [hunk 2]
```diff
@@ -1,33 +1,36 @@
 const DBHelper=require('./DB_helper');
 RunProgram();
 
+var database_name="rsquared_zilla_upazilla_db";
 async function RunProgram(){
     //bulkUpload();
```
Comment (azmainamin) [inline node.js_mongodb/app.js]: if function is not going to be used, remove the commented out line.

File: node.js_mongodb/DB_helper.js (hunk 1 above)
Comment (Shanzid01) [reply node.js_mongodb/DB_helper.js]: done.

File: node.js_mongodb/app.js (hunk 2 above)
Comment (Shanzid01) [reply node.js_mongodb/app.js]: done.

**Review by azmainamin — APPROVED** [review]

**Pull request merged**